IFC4x3/PropertyConstants/c/CONST_0JfPhvghHD3h$pIZ_hcMEX.xml

```diff
@@ -1,3 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="CONST_0JfPhvghHD3hpIZ_hcMEX" Name="CONST" UniqueId="13a59af9-aab4-4d0e-bff3-4a3fab9963a1" />
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="CONST_0JfPhvghHD3hpIZ_hcMEX" Name="CONST" UniqueId="13a59af9-aab4-4d0e-bff3-4a3fab9963a1">
+	<Documentation>Specifies that transition of the width between the previous width event and this event shall be constant and equal to the previously specified width meaning that the change in width shall occur instantaneously at the location of this event.</Documentation>
```

Comment:
This does not make sense. First, it says that the transition of the width is constant and finishes by saying that the change in width occurs instantaneously. It was probably meant that not the transition is constant but the width remains constant until it reaches the event in question.